### ISSUE TITLE
CI Restore bugs

### ISSUE THREAD

**Issue by dkasov** [Kentico/xperience-training-13#92]
### Brief bug description

I've cloned the master branch and tryed to restore the ci packages but i received an error saying:  The object type 'cms.document.medioclinic.basicpage' is missing the provider type configuration.
I need that to explore the builders module. maybe some ci xmls are wrong/missing in repo?
### Repro steps

2. Click on '...'
3. Scroll down to '...'

### Expected behavior

What is the correct behavior?

### Test environment

 - Platform/OS: [e.g. .NET Core 3.1, iOS, Windows]
 - Browser [e.g. Chrome, Safari, Edge Chromium, Edge]
 - Version [e.g. 81]

### Additional context

Add any other context about the problem here. If you found an issue where a similar bug has been reported, add a link here.

### Screenshots

Add links to screenshots, if you deem it appropriate.

### Issue expands upon

This issue expands upon another issue(s): #`issue number`

### Blockers

This issue is blocked by the following issue(s): #`issue number`

**Comment by eugene-cp** [comment]
I encountered the same problem during CI Restore, attached is my CI Log.

Version: K13/Hotfix 31
[ci.log](https://github.com/Kentico/xperience-training-13/files/6887057/ci.log)

**Comment by eugene-cp** [comment]
did some troubleshooting and this fixed the CI Restore for me:

- i modified @global\cms.documenttype\medioclinic.sitesection.xml 
- added a text field as follows:
`
     <field column="SiteSectionName" columntype="text" guid="09716bc5-a927-48f3-a08a-31dd101430fe" allowempty="true">
        <properties>
          <fieldcaption>SiteSectionName</fieldcaption>
        </properties>
      </field>
`

**Comment by JanLenoch** [comment]
Hi @eugene-cp and @dkasov ,

Thanks for reporting the issue and finding the missing node in the .xml file!

I tested it and I'll merge that right away.
